trie/utils/verkle.go

```diff
@@ -63,31 +63,34 @@ func GetTreeKey(address []byte, treeIndex *uint256.Int, subIndex byte) []byte {
 		var aligned [32]byte
 		address = append(aligned[:32-len(address)], address...)
 	}
-	var poly [5]fr.Element
 
-	poly[0].SetZero()
+	// poly = [2*64<<8, addressLE_low, addressLE_high, trieIndexLE_low, trieIndexLE_high]
+	var poly [5]fr.Element
```

Comment:
An extra helper comment to let future readers have a refresher on the EIP gist for the vector.